### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by trapacska**

### CODE REVIEW

**Comment by stevepeak** [comment]
Thank you @trapacska 

Suggested reviewers: @viktorbenei and/or @birmacher Thanks

**Comment by trapacska** [comment]
Sure, I'll ping @viktorbenei 🎉

File: step.yml [hunk 1]
```diff
@@ -35,4 +35,4 @@ inputs:
         Options to pass along to the Codecov uploader.
         You can use multiple options, separated by a space.
         Review all options at https://github.com/codecov/codecov-bash
-        Example: `-F unittests -J '^Project'`
```
Comment (viktorbenei) [inline step.yml]: revert this - see: https://github.com/bitrise-io/steps-codecov/pull/9

**Review by viktorbenei — CHANGES_REQUESTED** [review]

**Comment by viktorbenei** [comment]
Otherwise 👍

**Review by viktorbenei — APPROVED** [review]

File: bitrise.yml [hunk 2]
```diff
@@ -19,29 +19,37 @@ workflows:
         title: Step audit
         inputs:
         - content: stepman audit --step-yml=./step.yml
+    - script:
```
Comment (godrei) [inline bitrise.yml]: please do this, this way: https://github.com/bitrise-steplib/steps-xamarin-archive/blob/master/bitrise.yml#L32

**Review by godrei — APPROVED** [review]
👌

**Pull request merged**